internal/hammer/loadtest/hammer.go

```diff
@@ -31,27 +31,35 @@ type HammerOpts struct {
 	NumReadersRandom int
 	NumReadersFull   int
 	NumWriters       int
+	NumMMDVerifiers  int
+	MMDDuration      time.Duration
 }
 
 func NewHammer(tracker *client.LogStateTracker, f client.EntryBundleFetcherFunc, w LeafWriter, gen func() []byte, seqLeafChan chan<- LeafTime, errChan chan<- error, opts HammerOpts) *Hammer {
 	readThrottle := NewThrottle(opts.MaxReadOpsPerSecond)
 	writeThrottle := NewThrottle(opts.MaxWriteOpsPerSecond)
 
+	leafMMDChan := make(chan LeafMMD, opts.NumWriters)
+
 	randomReaders := NewWorkerPool(func() Worker {
 		return NewLeafReader(tracker, f, RandomNextLeaf(), readThrottle.TokenChan, errChan)
 	})
 	fullReaders := NewWorkerPool(func() Worker {
 		return NewLeafReader(tracker, f, MonotonicallyIncreasingNextLeaf(), readThrottle.TokenChan, errChan)
 	})
 	writers := NewWorkerPool(func() Worker {
-		return NewLogWriter(w, gen, writeThrottle.TokenChan, errChan, seqLeafChan)
+		return NewLogWriter(w, gen, writeThrottle.TokenChan, errChan, seqLeafChan, leafMMDChan)
+	})
+	mmdVerifiers := NewWorkerPool(func() Worker {
+		return NewMMDVerifier(tracker, errChan, leafMMDChan, opts.MMDDuration)
```

Comment:
nit: looking at this signature compared to the others, it stands out because the others have their channels passed in last. I'm not going to die on this hill, but it would be good if they were somewhat consistent.

Reply:
It's good to maintain the consistency. Rearranged the `NewMMDVerifier` arguments.